README.md

```diff
@@ -28,15 +28,92 @@ it('is a basic test with the page', async ({page}) => {
 });
 ```
 
-Run all tests with
+Run all tests across Chromium, Firefox and WebKit
 
 ```
 npx test-runner
 ```
 
-## Execution options
+Run tests on a single browser
+
+```
+npx test-runner --browser-name=chromium
+```
+
+Run all tests in headful mode
+
+```
+npx test-runner --headful
+```
+
+Take screenshots on failure
+
+```
+npx test-runner --screenshot-on-failure
+```
+
+See all options
+
 ```
 npx test-runner --help
 ```
 
+## Test fixtures
+TODO
+
+## Parameterize your tests
+A common pattern with end-to-end tests is to run test suites against multiple
+parameters.
+
+#### Default parameters
+`browserName`: Can be `chromium`, `firefox` or `webkit`. Tests are parameterized
+across the 3 browsers.
+
+#### Custom parameters
+You can define custom parameters
+
+```js
+import { fixtures } from '@playwright/test';
+
+// Define the parameter
+fixtures.defineParameter('appUrl', 'URL of the app to test against', 'http://localhost');
```

Comment:
I believe I'm missing a declaration for types, but unclear how that works. @pavelfeldman